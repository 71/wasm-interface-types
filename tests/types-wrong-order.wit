;; parse-fail

(module
  (@interface type (result s8) (param u8)))
