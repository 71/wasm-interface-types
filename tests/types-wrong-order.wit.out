expected keyword `func`
     --> ../../tests/types-wrong-order.wit:4:21
      |
    4 |   (@interface type (result s8) (param u8)))
      |                     ^
